src/feature/followUnfollow/__tests__/followUnfollowAction.test.js

```diff
@@ -0,0 +1,190 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import axios from 'axios';
+import {
+  getFollowing, followAuthor, unfollowAuthor, clearFollowing
+} from '../followUnfollowAction';
+import LocalStorage from '../../../__mocks__/localStorage';
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+const store = mockStore({});
+
+let storage;
+const token = 'Invalid token';
+describe('Follow Author Tests', () => {
+  beforeEach(() => {
+    moxios.install(axios);
+    storage = window.localStorage.setItem('token', token);
+    window.localStorage = new LocalStorage();
+  });
+  afterEach(() => {
+    moxios.uninstall(axios);
+    store.clearActions();
+    window.localStorage = storage;
+  });
+  test('Follow Author Tests', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: {
+          data: {
+            message: 'Successfully rated this article'
+          }
+        }
+      });
+    });
+    return store.dispatch(followAuthor()).then(() => {
+      expect(store.getActions().length).toEqual(1);
```

Comment:
@raymond42 while expecting that store.getActions().length equals to 1, a better and sure approach is to ascertain that the correct, a specific action has been dispatched. eg. store.getActions().toEqual(expectedAction). Kindly update where it is applicable in your tests.